yb-voyager/cmd/common.go

```diff
@@ -526,3 +528,75 @@ func getDefaultPGSchema(schema string) (string, bool) {
 		return "", true
 	}
 }
+
+func CleanupChildProcesses() {
+	myPid := os.Getpid()
+	processes, err := ps.Processes()
+	if err != nil {
+		log.Errorf("failed to read processes: %v", err)
+	}
+	childProcesses := lo.Filter(processes, func(process ps.Process, _ int) bool {
+		return process.PPid() == myPid
+	})
+	lo.ForEach(childProcesses, func(process ps.Process, _ int) {
```

Comment:
A simple `for` loop is better than the `lo.ForEach()`. The `ForEach()` is not worth it.